examples/redis/setup.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const { NodeTracer } = require('@opentelemetry/node');
+const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
+const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
+const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const EXPORTER = process.env.EXPORTER || '';
+
+function setupTracerAndExporters(service) {
+  const tracer = new NodeTracer({
+    plugins: {
+      http: {
```

Comment:
By default, `http` plugins is `enabled` with correct `path`, do you still want to add here? Once #503 merged, will add that into [default supported plugins list](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-node/src/config.ts#L29-L42).

Reply:
If I specify redis plugin here, will that "overwrite" the default list and only load the redis plugin and not any other default plugins? Ultimately this is all moot once redis becomes a default plugin, so I'm fine with blocking this until it becomes default

Reply:
Yes the `plugins` object overwrites the default, it does not merge with it. In any case, the redis plugin is now in the default list anyways so this doesn't matter.